README.md

```diff
@@ -24,7 +24,10 @@ webpack_encore:
     output_path: '%kernel.project_dir%/public/build'
     # If multiple builds are defined (as shown below), you can disable the default build:
     # output_path: false
-
+    
+    # if using Encore.enableIntegrityHashes() specify the crossorigin attribute value (default: anonymous)
```

Comment:
Also, do we really want to only add it when the `integrity` attribute is also present? Couldn't it make sense to also need the `crossorigin` attribute for other reasons?

Reply:
Correct. It is possible to have the assets on the same domain name as your site, and therefore integrity can be set without a crossorigin attribute. I have now made changes to support this by allowing the default/false to not show the attribute.